server/chat-commands/info.ts

```diff
@@ -819,15 +819,17 @@ export const commands: ChatCommands = {
 		if (!target) return this.parse('/help weakness');
 		if (!this.runBroadcast()) return;
 		target = target.trim();
-		const modName = target.split(',');
+		const modName = target.split(',').length > 1 ? target.split(',').pop() : null;
```

Comment:
This will _still_ cause the command to fail if you specify two things that don't include a modname.

For instance, `/weak fire, fighting`

Reply:
okay ill try another way